### DISCUSSION TITLE
Category filters inside modpacks

### DISCUSSION THREAD

**Discussion by rob-katt** [modrinth/.github#314] [Feature requests]
It would be a great quality-of-life improvement to have a filter option within the content list that uses the same category system as the mod search feature. This would be incredibly helpful when tweaking modpacks and would also streamline collaboration with others by making it easier to organize and manage specific types of mods. For instance, swapping out client-side mods like replacing JEI with REI due to personal preference would become much more straightforward.

<sub> Yes chat gpt was used to help me write this, because of dyslexia</sub>

**Comment by snewell92** [comment]
My use case for this feature, specifically for a "Client Side" filter in my instances' content list.

I created a modpack, in Modrinth, on my MBP, as I wanted to iron out any Metal/Apple/M-chip oddness and test it there. Then I exported an mrpack, got it on my windows machine, and ensured everything is fine on my AMD PC.

_Then_ I further copied it so I can host a local server for my family, and copied all the mods over into a new modded server managed by mcss, but now I have to go through and delete .jar files until the server doesn't yell at me about `DEDICATED_SERVER`.

Further - it would be nice to extract a server mr pack that does this for me! 🤩
